analyzers/tests/SonarAnalyzer.UnitTest/Facade/SyntaxFacadeTest.cs

```diff
@@ -87,12 +87,20 @@ public class SyntaxFacadeTest
             vb.NodeIdentifier(VB.SyntaxFactory.AttributeList()).Should().BeNull();
 
         [TestMethod]
-        public void NodeStringTextValue_UnexpectedType_CS() =>
-             cs.NodeStringTextValue(CS.SyntaxFactory.ThrowStatement(), null).Should().BeEmpty();
+        public void StringValue_UnexpectedType_CS() =>
+             cs.StringValue(CS.SyntaxFactory.ThrowStatement(), null).Should().BeNull();
 
         [TestMethod]
-        public void NodeStringTextValue_UnexpectedType_VB() =>
-            vb.NodeStringTextValue(VB.SyntaxFactory.ThrowStatement(), null).Should().BeEmpty();
+        public void StringValue_UnexpectedType_VB() =>
+            vb.StringValue(VB.SyntaxFactory.ThrowStatement(), null).Should().BeNull();
+
+        [TestMethod]
+        public void StringValue_NodeIsNull_ReturnsNull_CS() =>
```

Comment:
Not needed, obvious from assertion
```suggestion
        public void StringValue_NodeIsNull_CS() =>
```